lib/board.rb

```diff
@@ -0,0 +1,55 @@
+class Board
+  attr_accessor :cells
+
+  def initialize
+    cell_start
+  end
+
+  def reset!
+    cell_start
+  end
+
+  def cell_start
+    @cells = Array.new(9, " ")
+  end
```

Comment:
The delegation to the `cell_start` method here is unnecessary. You can just put this login in the `reset!` method, and call that method from `initialize`.